pipelines/qc/Snakefile

```diff
@@ -1,32 +1,69 @@
 import os
 import numpy as np
 import pandas as pd
-configfile: "config.yml"
 
-samples = pd.read_csv(config["manifest"], index_col=0)
-script_dir = config["scripts_loc"] 
+sampleset = os.path.split(os.getcwd())[1]
```

Comment:
What if we infer sample set from path to tracking and create the top-level directory for this set in the output `-d`?